### PR TITLE
8.0 ebp export without aeroo

### PR DESCRIPTION
**Pull request by sebastienbeau**
Add an ebp export version without Aeroo to solve this issue : https://github.com/OCA/l10n-france/pull/63

### CODE REVIEW

File: account_balance_ebp_csv_export/README.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Account Balance EBP CSV export
```
Comment (pedrobaeza) [inline account_balance_ebp_csv_export/README.rst:6]: Please use latest OCA's README: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: account_balance_ebp_csv_export/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+{
+    'name': 'Account Balance EBP CSV export',
+    'version': '8.0.0.0.1',
```
Comment (pedrobaeza) [inline account_balance_ebp_csv_export/__openerp__.py]: Use 8.0.1.0.0

**Comment by pedrobaeza** [comment]
A little remark, but :+1: (code review)

**Comment by legalsylvain** [comment]
Is it still WIP ?

**Comment by gurneyalex** [comment]
:+1:

**Comment by alexis-via** [comment]
I found a bug: the filtering by period or by date is not taken into account in the EBP CSV export ; the amounts given are the amounts without any date/period filter.

The bug was also present in the old version of the module "with aeroo".

**Comment by alexis-via** [comment]
FYI, I fixed the bug of the missing filter by date/period in the "aeroo" version of the module in this commit https://github.com/akretion/l10n-france/commit/5e8407c12e5ceeae274f76cc87a6bf5303fe17dc (it's in my branch 8.0-ebp-csv-import-aeroo where I keep the aeroo version for those that use it in production).

Also, it seems that the EBP CSV export should be encoded in latin1 and not in UTF-8. I made the change in the aeroo version and I'll have a confirmation when my customer will test it with Sage Etats Financiers.

**Comment by sebastienbeau** [comment]
@alexis-via Ok I will update the export with the change done in aeroo version

**Comment by alexis-via** [comment]
Seems to work nice with Teledec.fr.

**Comment by alexis-via** [comment]
I took into account Pedro's remarks.

We used this version of the module to declare Akretion's liasse fiscale via teledec today and it worked very well. So :+1:  and I merge.

**Pull request merged**